.github/workflows/ci.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     name: Ubuntu Jammy CI
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
```

Comment:
There should be more changes to this file:
```
on:
  pull_request:
  push:
    branches:
      - 'ign-sensors[0-9]'
      - 'gz-sensors[0-9]?'
      - 'main'
```

Reply:
added and force-pushed